test/controllers/criteria_controller_test.rb

```diff
@@ -0,0 +1,68 @@
+require 'test_helper'
```

Comment:
Getting one assertion, two errors on this one

Reply:
sorry, was merging. should work now

Reply:
Working!